### ISSUE TITLE
Karabiner Driver broken on Mac OS Big Sjur 11.2 Beta 2?

### ISSUE THREAD

**Issue by ogomez92** [pqrs-org/Karabiner-Elements#2581]
Hi,
I'm new to Karabiner and using the latest beta of Big Sur. The driver isn't working. I even did the uninstalll / reinstall process.

Can anyone confirm this?

**Comment by radiophage** [comment]
Same here

**Comment by Musk-Ox-Moo** [comment]
Yep, it's broken. There's a workaround. You have to disable system integrity protection from recovery mode.

Restart holding Command R, find Terminal in the Utilities, type **csrutil disable** and restart.

**Comment by hyunmindev** [comment]
Duplicate of #2548
